src/main/app/src/app/gebruikersvoorkeuren/zoekopdracht/zoekopdracht.component.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {MatDialog} from '@angular/material/dialog';
+import {GebruikersvoorkeurenService} from '../gebruikersvoorkeuren.service';
+import {Zoekopdracht} from '../model/zoekopdracht';
+import {Werklijst} from '../model/werklijst';
+import {Subscription} from 'rxjs';
+import {ZoekopdrachtSaveDialogComponent} from '../zoekopdracht-save-dialog/zoekopdracht-save-dialog.component';
+import {ZoekParameters} from '../../zoeken/model/zoek-parameters';
+import {ListParameters} from '../../shared/model/list-parameters';
+
+@Component({
+    selector: 'zac-zoekopdracht',
+    templateUrl: './zoekopdracht.component.html',
+    styleUrls: ['./zoekopdracht.component.less']
+})
+export class ZoekopdrachtComponent implements OnInit, OnDestroy {
+
+    @Input() werklijst: Werklijst;
+    @Input() zoekParameters: ZoekParameters | ListParameters;
+    @Output() zoekopdracht = new EventEmitter<Zoekopdracht>();
+    @Input() filtersChanged: EventEmitter<void>;
+
+    zoekopdrachten: Zoekopdracht[] = [];
+    actieveZoekopdracht: Zoekopdracht;
+    filtersChangedSubscription: Subscription;
```

Comment:
Ik dacht dat de naming convention is dat de naam van een subscription eindigd op een $ ? Ik zie dat wel op andere plaatsen in onze code

Reply:
yes, subject en subscription zie ik in de code dat we daar een $ voor gebruiken, emmitters dan weer niet